magellan-sample/src/androidTest/java/com/wealthfront/magellan/uitest/DialogTest.kt

```diff
@@ -22,12 +24,19 @@ class DialogTest {
     onView(withId(R.id.nextJourney)).perform(click())
 
     onView(withId(R.id.dialog)).perform(click())
-    assertShown { text("Hello") }
-    assertShown { text("Are you sure about this?") }
+
+    assertDialogContentsShown()
 
     onView(isRoot()).perform(orientationLandscape())
 
-    assertShown { text("Hello") }
-    assertShown { text("Are you sure about this?") }
+    assertDialogContentsShown()
   }
 }
+
+private fun assertDialogContentsShown() {
+  onView(withText("Hello"))
```

Comment:
I factored this out because thought I was going to need to target a dialog root for these matchers, but turns out I didn't have to...leaving it this 🤷